From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yuki Sireneva <yuki.utk8g@gmail.com>
Date: Sat, 4 Apr 2026 18:50:05 +0300
Subject: [PATCH] (Style) Increase maxSignatureSize

This patch is adapted from 64Gram.
Original author: Js0n <29531167+JasonKhew96@users.noreply.github.com>

diff --git a/Telegram/SourceFiles/ui/chat/chat.style b/Telegram/SourceFiles/ui/chat/chat.style
index 247b2f951a..94fe2aaeb5 100644
--- a/Telegram/SourceFiles/ui/chat/chat.style
+++ b/Telegram/SourceFiles/ui/chat/chat.style
@@ -171,7 +171,7 @@ maxStickerSize: 224px;
 maxAnimatedEmojiSize: 112px;
 maxGifSize: 320px;
 maxVideoMessageSize: 240px;
-maxSignatureSize: 144px;
+maxSignatureSize: 216px;
 maxWallPaperWidth: 160px;
 maxWallPaperHeight: 240px;
 historyThemeSize: size(272px, 176px);
